### ISSUE TITLE
chapter 2 is used twice

### ISSUE THREAD

**Issue by xentity** [vhwr/deezer_report#1]
2(a)
2(b)
.....

2(a)
2(b)
.....

**Comment by floridene** [comment]
yes, i know :) that's caused by the new chapter introdutcion. I will change the numbers for engineering part by time ;)
